entity_relation_joint_decoder.py

```diff
@@ -1,13 +1,21 @@
 from collections import defaultdict
 import json
 import os
-import random
 import logging
 import time
 
+import random
+random.seed(5216)
```

Comment:
seed 设置为全局变量吧，这样修改所有的 seed只需要改一处就行
```suggestion
SEED = 5216
random.seed(SEED)
```

Reply:
Done.